foreman_bootdisk.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
     'LICENSE',
     'README.md'
   ]
-  s.files = `git ls-files`.split("\n") - Dir['.*', 'Gem*', '*.gemspec']
+  s.files = `git ls-files`.split("\n") - Dir['.*', 'Gem*', '*.gemspec', 'gemfile.d/*']
```

Comment:
Do we need to distribute this with the gem? This will require packaging changes.

Reply:
As @ekohl said:
AFAIK gemfile.d is only used in tests and is the currently supported way to install additional dependencies. It's not in the gemspec and not used in packaging so I don't really see the problem.

We do this only, to run these tests but don't have a hard dependency for foreman_fog_proxmox in foreman_bootdisk.

Reply:
I agree with @lzap here that we don't need to ship it in the gem. We only use it in Jenkins and is really a custom solution that isn't used elsewhere AFAIK. It might just confuse people. What I would instead do is write something about this in README.